### ISSUE TITLE
error message when using all commands

### ISSUE THREAD

**Issue by Jacob26502** [ByteDream/crunchyroll-go#29]
Hi, I just tried installing this on my Ubuntu 21.04 VPS, and regardless of the command given, it always returns this message:

```admin@_______:~/crunchyroll-go$ crunchy
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/ByteDream/crunchyroll-go/v2/cmd/crunchyroll-go/cmd.systemLocale(0x0)
        /home/admin/crunchyroll-go/cmd/crunchyroll-go/cmd/utils.go:37 +0x35a
github.com/ByteDream/crunchyroll-go/v2/cmd/crunchyroll-go/cmd.init.0()
        /home/admin/crunchyroll-go/cmd/crunchyroll-go/cmd/archive.go:123 +0x45

**Comment by ByteDream** [comment]
This is an language detection error. crunchyroll-go assumes that the system language follows the scheme `language_country.UTF-8` (e.g. `en_US.UTF-8` or `de_DE.UTF-8`). 
What's the value of your `LANG` environment variable? (`echo $LANG`)

**Comment by Jacob26502** [comment]
it replies C.UTF-8

**Comment by ByteDream** [comment]
Fixed in release [v2.2.1](https://github.com/ByteDream/crunchyroll-go/releases/tag/v2.2.1).
